R-notebooks/dashboard_functions.R

```diff
@@ -0,0 +1,9 @@
+plot_28_day_frequency <- function(df_to_plot, geo_type) {
```

Comment:
It looks like you do a lot of the same processing steps before passing it to this function. Since plot.covidcast works on covidcast_signal objects, it might be cleaner to have a function that does the data manipulation, returns the data.frame and then just plot that.

Reply:
I refactored more common code in here based on later comment -- is this what you meant, or do you have another refactoring idea?

Reply:
I think my idea was slightly different, but this looks good